packages/core/src/components/forms/_helper-text.scss

```diff
@@ -0,0 +1,86 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+@import "../../common/mixins";
+
+/*
+Helper Text
+
+Add additional helper text to communicate user input. Additional markup is required for the more
+complicated DOM structure.
+
+- Use a `.pt-form-group` wrapper element.
+
+- Link the label and control elements with a `for={#id}` attribute on the `<label>` element and
+`id={#id}` on the corresponding control (only linked for the default example).
+
+- Add extra information to the label with `span.pt-text-muted`.
+
+- Add helper text to the form group to reflect user input (display intent with `.pt-helper-text.pt-intent-*`).
+
+Markup:
+<div class="pt-form-group {{.modifier}}">
+  <label for="example-form-group-input">
+    Label A
+    <span class="pt-text-muted">(optional)</span>
+  </label>
+  <div class="form-content">
```

Comment:
i was thinking about that - we don't have any css for it, but i do agree with keeping everything prefixed